packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.tsx

```diff
@@ -21,8 +21,8 @@ import IoArrowRightA from 'react-icons/lib/io/arrow-right-a';
 import { GrafanaTheme2, TraceKeyValuePair } from '@grafana/data';
 import { stylesFactory, withTheme2 } from '@grafana/ui';
 
+import { DURATION, NONE, TAG } from '../../../../public/app/core/components/SpanBarSettings/SpanBarSettings';
```

Comment:
In general it's not good idea to import things in package from core grafana as that can break how packages work (even though in case of jaeger-ui it may not be an issue). So I would rather keep the types that are needed in jaeger-ui inside the package and import them into the SpanBarSettings

Reply:
Just for context, packages like packages/grafana-ui are built into npm packages and published. If they import something from core grafana they work in grafana but will break for anybody using the package from NPM. We don't publish jaeger-ui but I think it's easier to just not do such imports and it may prevent some issues in the future.

Reply:
Done. :)